IPlug/APP/IPlugAPP_platform.h

```diff
@@ -0,0 +1,159 @@
+/*
+ ==============================================================================
+ 
+ This file is part of the iPlug 2 library. Copyright (C) the iPlug 2 developers.
+ 
+ See LICENSE.txt for  more info.
+ 
+ ==============================================================================
+ */
+
+#pragma once
+
+#include <IPlugPlatform.h>
+
+#include <string>
+#include <wdlstring.h>
+
+#ifdef OS_WIN
```

Comment:
Not sure if it's possible here but I don't like it in iPlug2 where we have e.g. a header file with both OS_WIN and OS_MAC implementation of the same class. If the interface is the same it's nicer if we can make a IPlugAPP_platform_mac.cpp and IPlugAPP_platform_win.cpp with the platform specific code. (Although more painful in terms of updating the examples)